judging-backend/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+enable_language(CXX)
+set(CMAKE_CXX_STANDARD 20)
+enable_language(C)
+set(CMAKE_C_STANDARD 11)
+find_package(Python3 REQUIRED COMPONENTS Interpreter)
```

Comment:
Does this search for `pypy3` or `python3`?

Reply:
It searches for `python3` by default. I don't know if CMake can search for `pypy3` so I just have an override in the judge file to use `pypy3` instead.